### PR TITLE
Use malayalam font for malayalam texts

### PR DESCRIPTION
**Pull request by shanavas786**
Malayalam texts hardly readable if supporting fonts are not installed in system

### CODE REVIEW

File: static/fonts/fonts.css [hunk 1]
```diff
@@ -16,3 +16,11 @@
     font-weight: normal;
     font-style: normal;
 }
+
+@font-face {
+  font-family: 'Manjari';
+  src: url('Manjari-Regular.woff2') format('woff2'),
```
Comment (asimqt) [inline static/fonts/fonts.css]: No fallback font?

**Comment by asimqt** [comment]
Can we add screenshots also?

File: static/fonts/fonts.css (hunk 1 above)
Comment (shanavas786) [reply static/fonts/fonts.css]: It will fallback to raleway font. no need for explicit declaration

**Comment by shanavas786** [comment]
With system font
sysdefault

With manjari
manjari

**Comment by biswaz** [comment]
We'd like to bring this PR back @shanavas786

**Comment by shanavas786** [comment]
@biswaz created https://github.com/IEEEKeralaSection/rescuekerala/pull/1114

Closing this in favour of #1114

**Pull request closed without merging**